compiler/ballerina-treegen/src/main/resources/syntax_tree_descriptor.json

```diff
@@ -3938,6 +3938,70 @@
                 }
             ]
         },
+        {
+            "name": "FunctionalMatchPatternNode",
+            "base": "Node",
+            "kind": "FUNCTIONAL_MATCH_PATTERN",
+            "attributes": [
+                {
+                    "name": "typeRef",
+                    "type": "Node"
+                },
+                {
+                    "name": "openParenthesisToken",
+                    "type": "Token"
+                },
+                {
+                    "name": "positionalArgMatchPatternsNode",
+                    "type": "Node",
+                    "occurrences": "MULTIPLE_SEPARATED"
+                },
+                {
+                    "name": "otherArgMatchPatternsNode",
+                    "type": "OtherArgMatchPatternsNode"
+                },
```

Comment:
The comma in between the two lists are not captured? 
Would it be better to merge these two lists in to one (similar to arguments list in a function call)?

Reply:
That comma is captured to the end of positionalArgMatchPatterns list.
Will look into that

Reply:
Changed